scripts/variantstore/wdl/extract/test_summarize_task_monitor_logs.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+from summarize_task_monitor_logs import parse_monitoring_log_files
+
+class TestSummarizeTaskMonitorLogs(unittest.TestCase):
+    def test_parse_monitoring_log_files(self):
+        import tempfile
+
+        input_files = ['summarize_task_monitor_logs_test_files/call-ScoreVariantAnnotationsINDELs/shard-34/monitoring.log',
```

Comment:
A few PEP 8 fouls re line lengths. 120 is the limit actually being [used for CAS](https://github.com/cellarium-ai/cellarium-cloud/blob/main/tox.ini#L13) which will likely be the "inspiration" for variantstore Python linting.

```
PEP 8: E501 line too long (123 > 120 characters)
```

Reply:
Got it.